apps/synapse-portal-framework/src/components/elportal/ELBrowseToolsPage.tsx

```diff
@@ -0,0 +1,292 @@
+import { SynapseComponents, FeaturedToolsList } from 'synapse-react-client'
+import Layout from '../Layout'
+import React from 'react'
+import { Box, Button, Link, TextField, Typography } from '@mui/material'
+import { Query, TextMatchesQueryFilter } from '@sage-bionetworks/synapse-types'
+import PopularSearches from '../PopularSearches'
+import pluralize from 'pluralize'
+import Ecosystem from '../csbc-home-page/Ecosystem'
+
+type Category = {
+  resourceName: string
+}
+
+const categories: Category[] = [
+  { resourceName: 'Cell Lines' },
+  { resourceName: 'Gene Targets' },
+  { resourceName: 'Biomarkers' },
+  { resourceName: 'Chemoinformatics' },
+]
+
+export type ELBrowseToolsPageProps = {
+  popularSearchesSql: string
+  toolsSql: string
+}
+
+const ELBrowseToolsPage = (props: ELBrowseToolsPageProps) => {
+  const { popularSearchesSql, toolsSql } = props
+  const [searchText, setSearchText] = React.useState<string>('')
+  const gotoExploreTools = () => {
+    window.location.assign('/Explore/Computational%20Tools')
+  }
+
+  const gotoExploreToolsWithSelectedResource = (selectedResource: string) => {
```

Comment:
There is no (faceted) column that contains these categories in the Computational Tools table (yet).  So this will not work until the data is updated.